components/scream/src/physics/p3/atmosphere_microphysics.hpp

```diff
@@ -83,6 +83,79 @@ class P3Microphysics : public scream::AtmosphereProcess
 
 }; // class P3Microphysics
 
+/*
+ * This class is microphysics stub meant to be used for initialization of
+ * stand-alone tests.  It is similar to P3Microphysics except that it won't
+ * actually calculate anything, it will only initialize the input fields needed
+ * by atmophseric microphysics.
+ * TODO: Switch to a general stand-alone class for initialization of any stand-alone
+ * test.
+*/
+class SAMicrophysics : public scream::AtmosphereProcess
```

Comment:
De gustibus, but you could rename the class to something more meaningfull, like `P3InputsProvider` or something like that. This process, per se, does not implement microphysics, but rather simply makes sure P3 inputs are valid.

Reply:
Yes, that could certainly work.  I'll make the change.